_includes/configuration/content-langs.md

```diff
@@ -0,0 +1,87 @@
+{% if page.name == "tinymcespellchecker.md" %}
+### `content_langs`
+{% else %}
+## `content_langs`
+{% endif %}
+
+{{ site.requires_5_9v }}
+
+This option sets the languages displayed in the `language` dropdown toolbar button and the `language` menu item. The option accepts an array of language objects. Each language object is a JavaScript object defined using the following properties.
+
+| Name | Type | Requirement | Description |
+| ---- | ---- | ----------- | ----------- |
+| `title` | String | required | The name shown for this language in the UI. |
+| `code` | String | required | The language tag, as defined in [Tags for Identifying Languages (BCP47)](https://www.ietf.org/rfc/bcp/bcp47.txt), for this language. This tag is placed in the `lang` attribute for elements formatted using this language. |
+| `customCode` | String | optional | An alternative language tag, that does not need to be compatible with BCP47. This code is placed in the `data-mce-lang` attribute for elements formatted using this language, if it is defined. |
+
+**Type:** `Array`
+
+{% if page.name == "tinymcespellchecker.md" %}
+**Default:**
+
+```js
+[
+    { title: 'English', code: 'en' },
+    { title: 'Spanish', code: 'es' },
+    { title: 'French', code: 'fr' },
+    { title: 'German', code: 'de' },
+    { title: 'Portuguese', code: 'pt' },
+    { title: 'Chinese', code: 'zh' }
+]
+```
+
+> **Note:** This default is provided by the Spell Checker Pro plugin, it is not available in the core editor.
+
+> **Note:** The languages specified by the deprecated [`spellchecker_select_language`](#spellchecker_select_language) option are used as a fallback if they are available.
```

Comment:
Having 2 notes like this reads and looks very strange. Can we merge this into the one note, as it's all related?